Resources/doc/development/breadcrumb.md

```diff
@@ -1,77 +1,95 @@
 # Using the breadcrumb
```

Comment:
die uitleg is mij niet echt duidelijk, mss voorbeeld bijzetten hoe de cronjob er precies uitzien en beter uitleggen hoe het werkt?
alsook hoe werken vertalingen?

Reply:
@jonasdekeukelaere extra info toegevoegd ivm vertalingen 358de4dd83f39e3ada3a7d918e252f7864f60e11
ivm met cronjobs geen idee wat je bedoeld, heeft hier niks mee te maken

Reply:
breadcrumbs* (geen idee waarom ik cronjobs zei)